### DISCUSSION TITLE
Monitoring Client/Server IP addresses

### DISCUSSION THREAD

**Discussion by Mohit-3196** [ant-media/Ant-Media-Server#5002] [FAQs]
I'm using Grafana monitoring tools to monitor my Ant Media Server and would like to know the IP addresses of the Client/Server.
Can you please tell me if it is possible and if so, then how can I get these stats?

Thank you.

**Comment by zerostack0** [comment]
Hi @Mohit-3196 
Yes, it's possible. You can get the IP address by getting stats using [Rest API](https://antmedia.io/rest/#/BroadcastRestService/getWebRTCClientStatsListV2) and you can see it from the Grafana dashboard.
